api/lib/infrastructure/repositories/organization-repository.js

```diff
@@ -104,10 +104,10 @@ module.exports = {
       });
   },
 
-  findBy(filters) {
+  findByExternalIds(externalIds) {
     return BookshelfOrganization
-      .where(filters)
-      .fetchAll()
+      .where('externalId', 'in', externalIds)
```

Comment:
Il me semble qu'on utilise `whereIn` d'habitude : 
```
.whereIn('externalId', externalIds)
```

Reply:
`whereIn` est une méthode de knex.js. Ici on est sur un objet Bookshelf, et il n'ont pas implémenter le `whereIn`. Donc on utilise le `where` avec la clause `in` :)

Reply:
Tu pouvais aussi faire : `.query({ whereIn: externalIds })` (avec `.query` tu peux demander à Bookshelf d'appeler n'importe quelle méthode `knex`). Mais si le `where(…,'in',…)` fonctionne c'est probablement bien aussi.